code/modules/mob/living/carbon/human/species/station/station_vr.dm

```diff
@@ -331,3 +331,40 @@
 	gluttonous = 0
 	spawn_flags = SPECIES_CAN_JOIN | SPECIES_IS_WHITELISTED
 	min_age = 18
+
+datum/species/harpy
+	name = "Rapala"
+	name_plural = "Rapalans"
+	icobase = 'icons/mob/human_races/r_harpy_vr.dmi'
+	deform = 'icons/mob/human_races/r_def_harpy_vr.dmi'
+	tail = "tail"
+	icobase_tail = 1
+	unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/claws, /datum/unarmed_attack/bite/sharp)
+	num_alternate_languages = 3
+	secondary_langs = list(LANGUAGE_BIRDSONG)
+	name_language = null
+	color_mult = 1
+
+	min_age = 18
+	max_age = 80
+
+	blurb = "An Avian species, coming from a distant planet, the Rapalas are the very proud race.\
```

Comment:
There's some pretty big mistakes in this blurb. It needs thought out and written properly.

Reply:
What mistakes are you talking about, though that does remind me i should remove some of the attacks

Reply:
Something more like:
```blurb = "An Avian species, coming from a distant planet, the Rapalans are a very proud race.\
Sol researchers have commented on them having a very close resemblance to the mythical race called 'Harpies',\
who are known for having massive winged arms and talons as feet. They've been clocked at speeds of over 35 miler per hour chasing the planet's many fish-like fauna.\
The Rapalan's home-world 'Verita' is a strangely habitable gas giant, while no physical earth exists, there are fertile floating islands orbiting around the planet from past asteroid activity."```

Would be better. There were quite a few grammar mistakes and such in there.

Reply:
Thank you im very poor at catching those kinds of things in other people's work and i thought i had got them all with a revision someone else provided